docs/development-setup.md

```diff
@@ -57,6 +57,14 @@ adb reverse tcp:8000 tcp:8000
 12. Take a look at the style by viewing the test map: [http://localhost:5002](http://localhost:5002)
 13. Take a look at the backend: [http://localhost:8000](http://localhost:8000) (The public version is available at
     api.entitlementcard.app)
+14. Set up the matomo instance [http://localhost:5003](http://localhost:5003) (The public version is available at https://matomo-entitlementcard.tuerantuer.org)
+15. (optional) Add your matomo config to the backend config
+```yaml
+matomo:
+  siteId: 1
+  url: http://localhost:5003
+  accessToken: <matomo-access-token>
```

Comment:
how can i find the token?

Reply:
visit `localhost:5003` and go to the settigns. Either search for "token" or go to "Persönlich->Sicherheit"

Reply:
i had to adjust the `config.ini.php` on the docker container to changed the trusted hosts and add the port
Not sure if that is just my problem
```js
[General]
salt = "36941bec3f510a9c9411000c89ee712b"
trusted_hosts[] = "localhost:5003"
```

Reply:
@sarahsporck please add how to findthe `matomo-access-token` and update the `url` to `http://localhost:5003/matomo.php`